### PR TITLE
Upgrade version 0.9.3

### PR DESCRIPTION
**Pull request by nohkwak**
- Signer Bug Fix 
- Remove option for setting wallet dynamicUpdate up

### CODE REVIEW

File: ethers-ext/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@klaytn/ethers-ext",
-  "version": "0.9.2-beta",
+  "version": "0.9.3",
```
Comment (kjeom) [inline ethers-ext/package.json:3]: 0.9.3 is also in beta

File: web3py-ext/setup.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_packages
 NAME = "web3py_ext"
-VERSION = "0.9.2-beta"
+VERSION = "0.9.3"
```
Comment (kjeom) [inline web3py-ext/setup.py:3]: please check above comment

File: ethers-ext/package.json (hunk 1 above)
Comment (nohkwak) [reply ethers-ext/package.json]: Updated

File: web3py-ext/setup.py (hunk 2 above)
Comment (nohkwak) [reply web3py-ext/setup.py]: Good, thanks.

**Comment by kjeom** [comment]
@JayChoi1736 It will not affect web3j-ext.

**Review by kjeom — APPROVED** [review]

**Comment by JayChoi1736** [comment]
> @JayChoi1736 It will not affect web3j-ext.

Okay :)

should we consider managing version infos of each extensions in one place?

**Comment by kjeom** [comment]
> > @JayChoi1736 It will not affect web3j-ext.
> 
> Okay :)
> 
> should we consider managing version infos of each extensions in one place?

@JayChoi1736 Let's discuss before publishing v1.0.0

**Pull request merged**